contracts/PreseasonTwo/BuilderNFTPreSeason02Implementation.sol

```diff
@@ -396,6 +396,29 @@ contract BuilderNFTPreSeason02Implementation is
     function minter() external view returns (address) {
         return MemoryUtils._getAddress(MemoryUtils.MINTER_SLOT);
     }
+    function mintTo(
+        address account,
+        uint256 tokenId,
+        uint256 amount
+    ) external onlyAdminOrMinter {
+        _validateMint(account, tokenId);
+        _mintTo(account, tokenId, amount);
+    }
+
+    function _mintTo(
```

Comment:
We should also replace the last 2 lines of mint() function with this method, that way we have standard minting behaviour